logging_demo/src/logger_config_component.cpp

```diff
@@ -0,0 +1,80 @@
+// Copyright 2017 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "logging_demo/logger_config_component.hpp"
+
+#include <cinttypes>
+#include <iostream>
+#include <memory>
+
+#include "logging_demo/srv/config_logger.hpp"
+#include "rclcpp/rclcpp.hpp"
+#include "rcutils/error_handling.h"
+
+namespace logging_demo
+{
+
+LoggerConfig::LoggerConfig()
+: Node("logger_config")
+{
+  auto handle_logger_config_req =
+    [this](
```

Comment:
You only ever appear to use the `get_name()` of `this`, maybe just capture the name?

Reply:
Or make it a method like you did elsewhere in this pr.